lib/awful/mouse/init.lua

```diff
@@ -297,8 +297,8 @@ function mouse.object.set_current_client() end
 
 function mouse.object.get_current_wibox()
     local obj = capi.mouse.object_under_pointer()
-    if type(obj) == "drawin" then
-        return obj
+    if type(obj) == "drawin" and obj.wibox then
+        return obj.wibox()
```

Comment:
Ok, so this confirms that I misremembered this when writing the above comment. Just make sure this works (and the code change is in the right commit) and merge it then. Sorry for my confusion.